packages/react-native-bpk-component-badge/src/BpkBadge.js

```diff
@@ -30,9 +30,9 @@ import {
 import {
   colorGray900,
   colorGray50,
```

Comment:
all the grays need changing too, but it's not a visual change so lets do that in another PR 👍